sample/replication/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ volumes:
 
 services:
   pg-master:
-    image: 'kartoza/postgis:manual-build'
+    image: 'kartoza/postgis:13.0'
```

Comment:
I propose that we keep the image tag used here as `manual-build` (and in the logical_replication as well). 
The main advantage would be:
- Encourage first time user/tester to build first against current repo
- We don't have to change the recipe everytime we upgrade to next major version.

If user wants to use specific target image (12 or 13), they can always do that via overriding the `docker-compose.override.yml` and specify the tag.

But it's also harmless if you specify the tag, so I have neutral opinion on this.

Reply:
I though if we wanted to let users test the replication then it would be an easier way to test against official builds as it would give us feedback if all is working rather than having them to build which I feel scares a few people out there